etc/kayobe/containers/cron/files/logrotate.conf

```diff
@@ -0,0 +1,32 @@
+weekly
+
+rotate 3
+
+copytruncate
+
+compress
+
+delaycompress
+
+notifempty
+
+missingok
+
+minsize 30M
+
+maxsize 100M
+
+su root kolla
+
+"/var/log/kolla/ironic/*.log"
+{
+}
+"/var/log/kolla/ironic-inspector/*.log"
+{
+}
+"/var/log/kolla/nginx/*.log"
+{
+}
+"/var/log/kolla/squid/*.log"
```

Comment:
Little bit strange have this one under /var/log/kolla since it isn't a kolla service. Could we just bind mount it to /var/log/squid instead? Possibly needs different setting in place of `su root kolla`?